postman/collections/mm2_dev.postman_collection.json

```diff
@@ -4824,6 +4824,45 @@
 						}
 					]
 				},
+				{
+					"name": "get_current_mtp",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"// Strip JSON Comments\r",
+									"const rawData = pm.request.body.toString();\r",
+									"const strippedData = rawData.replace(/\\\\\"|\"(?:\\\\\"|[^\"])*\"|(\\/\\/.*|\\/\\*[\\s\\S]*?\\*\\/)/g, (m, g) => g ? \"\" : m)\r",
+									"\r",
+									"pm.request.body.update(strippedData);"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"userpass\": \"{{userpass}}\",\n    \"mmrpc\": \"2.0\",\n    \"method\": \"get_current_mtp\",\n    \"params\": {\n        \"coin\": \"RICK\"\n    },\n    // \"id\": null // Accepted values: Integers\n}\n"
```

Comment:
The request doesn't work because of the trailing comma at the end here, please remove this comma.
`    "params": {
        "coin": "RICK"
    },`

Reply:
done